public/scripts/open-project.js

```diff
@@ -0,0 +1,115 @@
+(function() {
+  require(["jquery"], function($) {
+    var projects = document.querySelectorAll("tr.bramble-user-project");
+
+    function generateUrl(path, date) {
+      var url = window.location;
+      var qs = url.search;
+      if(qs.length < 1) {
+        qs = "?";
+      }
+      qs += "now=" + encodeURIComponent(date);
+
+      return url.protocol + "//" + url.host + "/newProject/" + encodeURIComponent(path) + qs;
+    }
+
+    $("#project-submit").on("click", function(e) {
+      e.preventDefault();
+
+      var projectName = document.getElementById("project-name");
+      var value = projectName.value;
+
+      if (value.length < 1) {
+        return;
+      }
+
+      var request = $.ajax({
+        type: "GET",
+        url: "/projectExists/" + encodeURIComponent(value),
+        statusCode: {
+          404: function() {
+            if (request.status === 404) {
```

Comment:
Why do you do this check?  Isn't the point of doing `statusCode: { 404: function() {...} }` so that you know it's that code?